upup/pkg/fi/cloudup/apply_cluster.go

```diff
@@ -643,6 +649,79 @@ func validateDNS(cluster *api.Cluster, cloud fi.Cloud) error {
 		glog.V(2).Infof("Found NS records for %q: %v", dnsName, hosts)
 	}
 
+	// TODO: Move to update
+	if os.Getenv("KOPS_FF_SKIP_DNS_PRECREATE") == "" {
```

Comment:
I know it's a WIP - so don't mind me snooping :) 

Are we keeping the `$ENVAR`'s in kops? If so just curious if we can/should start a manifest of all of them, with their behavior?